templates/Umbraco.Templates.csproj

```diff
@@ -13,30 +13,30 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <Content Include="UmbracoPackage\**" Exclude="bin;obj" />
-    <Content Include="UmbracoPackageRcl\**" Exclude="bin;obj" />
-    <Content Include="UmbracoProject\**" Exclude="bin;obj" />
-    <Content Include="..\src\Umbraco.Web.UI.New\Program.cs">
+    <Content Include="..\src\Umbraco.Web.UI\Program.cs">
       <Link>UmbracoProject\Program.cs</Link>
       <PackagePath>UmbracoProject</PackagePath>
     </Content>
-    <Content Include="..\src\Umbraco.Web.UI.New\Views\Partials\blocklist\**">
+    <Content Include="UmbracoPackage\**" Exclude="bin;obj" />
+    <Content Include="UmbracoPackageRcl\**" Exclude="bin;obj" />
+    <Content Include="UmbracoProject\**" Exclude="bin;obj" />
+    <Content Include="..\src\Umbraco.Web.UI\Views\Partials\blocklist\**">
       <Link>UmbracoProject\Views\Partials\blocklist\%(RecursiveDir)%(Filename)%(Extension)</Link>
       <PackagePath>UmbracoProject\Views\Partials\blocklist</PackagePath>
     </Content>
-    <Content Include="..\src\Umbraco.Web.UI.New\Views\Partials\grid\**">
+    <Content Include="..\src\Umbraco.Web.UI\Views\Partials\grid\**">
```

Comment:
The Umbraco.GridLayout editor has been removed, so we don't need the old partials anymore either. They fail the build if you install it through Umbraco.Templates since they reference assemblies that no longer exist.